src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +34,7 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    // Created new method for access to newly-changed private variable "feedbackToUser"
```

Comment:
Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).